source/parser.hera

```diff
@@ -151,6 +158,9 @@ UpdateExpressionSymbol
 
 # https://262.ecma-international.org/#prod-AssignmentExpression
 AssignmentExpression
+  PipelineExpression
+  # TODO If NonPipelineAssignmentExpression or SingleLineAssignmentExpression is used here then behavior changes
```

Comment:
@STRd6 I am not entirely sure why I can't use the SingleLineAssignmentExpression rule here to remove the duplication. Somehow how the parser parses call expressions changes if I do that.

Reply:
I've run into similar trouble trying to simplify this as well. It's not necessary to have it for this PR, but hopefully one of us can figure it out at some point.